### PR TITLE
EbayVideo: improvements

### PR DESCRIPTION
**Pull request by darkwebdev**
implements #188 
- add missing key attribute in sources.map
- fix `undefinedpx` when width/height props are not passed
- allow to hide report button
- show captions button instead of menu?
- get rid of `any` type

### CODE REVIEW

File: src/ebay-video/__tests__/index.stories.tsx [hunk 1]
```diff
@@ -1,18 +1,21 @@
 import React, { useState } from 'react'
 import { Meta } from '@storybook/react'
-import { EbayButton, EbayVideo, EbayVideoSource } from '../../index'
+import 'shaka-player/dist/controls.css'
```
Comment (HenriqueLimas) [inline src/ebay-video/__tests__/index.stories.tsx:3]: We need to add this info in the readme also that applications needs to import this css

File: src/ebay-video/const.ts [hunk 2]
```diff
@@ -11,8 +11,8 @@ export const defaultVideoConfig = {
         'spacer',
         'mute',
         'report',
-        'fullscreen',
-        'overflow_menu'
-    ],
-    overflowMenuButtons: ['captions']
+        'captions',
+        // 'quality',
```
Comment (HenriqueLimas) [inline src/ebay-video/const.ts:15]: Do we need this comment?

Comment (darkwebdev) [reply src/ebay-video/const.ts:15]: added comment for this

File: src/ebay-video/__tests__/index.stories.tsx (hunk 1 above)
Comment (darkwebdev) [reply src/ebay-video/__tests__/index.stories.tsx:3]: added

**Review by HenriqueLimas — APPROVED** [review]

**Pull request merged**